processing/src/test/java/org/apache/druid/granularity/QueryGranularityTest.java

```diff
@@ -802,17 +800,6 @@ private void assertSameInterval(List<DateTime> expected, Iterable<Interval> actu
     Assert.assertFalse("actualIter not exhausted!?", actualIter.hasNext());
     Assert.assertFalse("expectedIter not exhausted!?", expectedIter.hasNext());
   }
-
-  @Test(timeout = 60_000L)
-  public void testDeadLock() throws Exception
```

Comment:
IIRC this was a regression test, so it'd be nice to keep it in (although I suppose not essential, since the likelihood of the bug reoccurring is probably pretty low). Maybe creating the `JvmUtils.systemClassPath()` method would make this test easier to keep in.

Reply:
I thought about it, but I tried to explain why I think this test does not add much value in the PR description. If we feel strongly about it though, we can try to make it work.

Reply:
Sorry, I missed that part of the description. I do think it's fine to remove the test at this point.